Process.c

```diff
@@ -821,34 +836,13 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
 
       xSnprintf(buffer, n, "%4ld ", this->nlwp);
       break;
-   case PERCENT_CPU:
+   case PERCENT_CPU: Process_printPercentage(this->percent_cpu, buffer, n, &attr); break;
```

Comment:
Line wrapping and indent:
```suggestion
   case PERCENT_CPU:
      Process_printPercentage(this->percent_cpu, buffer, n, &attr);
      break;
```

Reply:
The case statement is formatted mixed already, and since we don't need to touch this line right now I'd favor to leave it alone at this time. We may decide to have a complete reformat of this (and related) switch statements, but IMHO this would introduce too much noise right now.